helix-term/Cargo.toml

```diff
@@ -61,5 +61,9 @@ serde = { version = "1.0", features = ["derive"] }
 grep-regex = "0.1.9"
 grep-searcher = "0.1.8"
 
+# compiling grammars
+cc = { version = "1" }
+threadpool = { version = "1.0" }
```

Comment:
How about housing the grammar code in a `helix-loader` crate?

Reply:
Ah good idea, it seems pretty out of place in helix-term.